extensions/2.0/Vendor/EXT_structural_metadata/README.md

```diff
@@ -11,40 +13,490 @@
+>             ]
+>           }
+>         }
+>       }
+>     }
+>   }
+> }
+> ```
+
+#### Enum Value
+
+*Defined in [enum.value.schema.json](./schema/enum.value.schema.json).*
+
+Pairs of `(name, value)` entries representing possible values of an enum property.
+
+Enum values are defined as entries in the `enum.values` array. Duplicate names or duplicate integer values are not allowed.
+
+
+
+
+## Metadata Storage
+
+The classes that defined in the schema are templates describing the data types and meanings of properties. An instance of such a metadata class is referred to as a _metadata entity_, and can be created from a set of values that conform to the structure of the class. This extension defines different ways of storing large amounts of property values inside a glTF asset, in compact binary forms: 
+
+- **Property Tables** store property values as numeric arrays in a parallel, column-based binary layout, using standard glTF buffer views. These tables can be accessed with a row index, and allow associating complex, structured metadata with arbitrary types with entities of a glTF asset on different levels of granularity.
+- **Property Mappings** are a way of storing metadata as vertex attributes, using standard glTF accessors. They can be used to associate certain forms of metadata with vertices of a mesh primitive. 
+- **Property Textures** store property values in channels of a texture, suitable for very high-frequency data mapped to less-detailed 3D surfaces. 
+
+The following sections describe these storage formats in more detail.
+
+### Property Tables
+
+*Defined in [propertyTable.schema.json](./schema/propertyTable.schema.json).*
+
+Each property table defines a specified number (`count`) of metadata entities conforming to a particular class (`class`), with property values stored as parallel arrays in a column-based binary layout. Property tables support a richer variety of data types than glTF accessors or GPU shading languages allow, and are suitable for datasets that can be expressed in a tabular layout.
+
+Property tables are defined as entries in the `propertyTables` array of the root-level `EXT_structural_metadata` extension, and may be referenced by other extensions. 
+
+The property table may provide value arrays for only a subset of the properties of its class, but class properties marked `required: true` must not be omitted. Each property value array given by the property table must be defined by a class property with the same alphanumeric property ID, with values matching the data type of the class property.
+
+> **Example:** A `tree_survey_2021-09-29` property table, implementing the `tree` class defined in earlier examples. The table contains observations for 10 trees, with each property defined by a buffer view containing 10 values.
+>
+> ```jsonc
+> {
+>   "extensions": {
+>     "EXT_structural_metadata": {
+>       "schema": { ... },
+>       "propertyTables": [{
+>         "name": "tree_survey_2021-09-29",
```

Comment:
Why was it removed?

These are "complex" objects that store application-specific data. I could imagine that there will sometimes be the necessity to present _some_ information about them in the UI: "This table stores the data that was obtained via survey 2021-09-29", vs. "This one is for 2021-09-30" ...

(The question is remotely related to the questions about "tags" for "Feature ID sets": How will the user select the data that is visualized, and how will he _know_ what this data _is_?).

I think that at least having the _option_ to transport some "application/domain-specific information" there could be desirable. This could be a plain `name`, or a `description`. If it is only a name, this could, by the way, also be accomplished by not extending `glTFProperty`, but [`glTFChildOfRootProperty`](https://github.com/javagl/glTF/blob/60efccc60e05e5764a48722a37d4a0a5fae46aac/specification/2.0/schema/glTFChildOfRootProperty.schema.json).

Reply:
(Unless that "...ChildOfRoot..." is not considered to be not-fitting, because it's rather a "...ChildOfExtension..." :-/ )

Reply:
Those are good points, some application/domain-specific information is helpful. I'll add `name` back.

Reply:
My original thinking was that a feature ID set plus class name and description was sufficient to describe the data, but now I agree that property table `name` is useful too.

Reply:
Pushed the schema changes:

https://github.com/CesiumGS/glTF/commit/9486ece8a3d33bf243f0c06c0cce9718a574b14c
https://github.com/CesiumGS/3d-tiles/commit/1746ca99df2a50a809a984f917fd8a7011579354